libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -1004,10 +1004,6 @@ Literal::SubDenomination ASTJsonImporter::subdenomination(Json::Value const& _no
 		return Literal::SubDenomination::Wei;
 	else if (subDenStr == "gwei")
```

Comment:
`gwei` seems to exist here, but not in the frontend?

Reply:
Someone needs to do the clean up change for the `breaking` branch (cc @Marenz), but `gwei` was added to 0.6.11 in a non-breaking way (e.g. without tokens).